src/bin/gpfusion/gpbridgeapi.c

```diff
@@ -536,3 +536,8 @@ void free_token_resources(PxfInputData *inputData)
 
 	pfree(inputData->token);
 }
+
+void free_dfs_address()
+{
+	free(dfs_address);
+}
```

Comment:
Yes only in the secure mode. However it is only dispatched from master in the secure mode as well so it is never allocated in any other case